src/common/elastic.service.ts

```diff
@@ -97,27 +97,23 @@ export class ElasticService {
   }
 
   async getTokensByIdentifiers(identifiers: string[]) {
-    const elasticQueryAdapter: ElasticQuery = new ElasticQuery();
-    elasticQueryAdapter.condition.should = identifiers.map(identifier => 
+    const queries = identifiers.map(identifier => 
       QueryType.Match('identifier', identifier, QueryOperator.AND)
     );
 
-    const elasticQuery = buildElasticQuery(elasticQueryAdapter);
+    const elasticQueryAdapter = ElasticUtils.boilerplate(QueryConditionOptions.should, queries);
+    const elasticQuery = ElasticUtils.buildElasticIndexerQuery(elasticQueryAdapter);
 
     let documents = await this.getDocuments('tokens', elasticQuery);
 
     return documents.map((document: any) => this.formatItem(document, 'identifier'));
   }
 
   async getAccountEsdtByIdentifiers(identifiers: string[]) {
-    const elasticQueryAdapter: ElasticQuery = new ElasticQuery();
-    elasticQueryAdapter.condition.should = identifiers.map((identifier) => QueryType.Match('identifier', identifier, QueryOperator.AND));
-    
-    let pagination = new ElasticPagination();
-    pagination.size = 10000;
-    elasticQueryAdapter.pagination = pagination;
+    const queries = identifiers.map((identifier) => QueryType.Match('identifier', identifier, QueryOperator.AND));
 
-    const elasticQuery = buildElasticQuery(elasticQueryAdapter);
+    const elasticQueryAdapter = ElasticUtils.boilerplate(QueryConditionOptions.should, queries, { from: 0, size: 10000 });
+    const elasticQuery = ElasticUtils.buildElasticIndexerQuery(elasticQueryAdapter);
```

Comment:
this is also boilerplate. The getDocuments function below should accept an ElasticQuery parameter directly, and if it's ever used directly as an object, maybe create a `getDocumentsRaw` function to be lower-level